src/actions/motion.ts

```diff
@@ -615,6 +615,12 @@ class MarkMovementBOL extends BaseMovement {
       await ensureEditorIsActive(mark.document);
     }
 
+    const editor = vscode.window.activeTextEditor;
+    if (!editor) {
+      throw VimError.fromCode(ErrorCode.MarkNotSet);
+    }
+    vimState.editor = editor;
```

Comment:
Unfortunately setting `VimState.editor` directly is misguided. It's not `readonly`, but it should be.